config/gulp/javascript.js

```diff
@@ -27,6 +27,42 @@ gulp.task('eslint', function (done) {
 
 });
 
+gulp.task('copy-docs-javascript', function (done) {
+
+  dutil.logMessage(task, 'Copying JS from js/');
+
+  var stream = gulp.src('./js/**/*')
+    .pipe(gulp.dest('assets/js/'));
+
+  return stream;
+
+});
+
+gulp.task('copy-jquery-javascript', function (done) {
```

Comment:
Does this need to be a task or can we just have it as part of the `package.json` in this repo?